src/cljc/centrebull/spec.cljc

```diff
@@ -74,7 +74,7 @@
 ;Clojure spec predicate for a non empty string
 (def non-empty-string (s/and string? #(not= "" %)))
 
-(defn- str->int [s] #?(:cljs (cond (and (string? s) #(not= "" %)) (cljs.tools.reader/read-string s)
+(defn- str->int [s] #?(:cljs (cond (and (string? s) #(not= "" %)) s
```

Comment:
The cljs.tools/reader reads a string and attempts to convert it into a primative type. i.e. "1234" becomes 1234.